src/clientapi/index.js

```diff
@@ -118,18 +118,25 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   import .UI;
   this.ui = new UI();
 
-
-  // this.track({
-  //  name: "campaignID",
-  //  category: "campaign",
-  //  subcategory: "id",
-  //  data: campaign
-  // });
-
   var map;
   try {
     if (GLOBAL.CACHE) {
       map = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+
+      // Add some defaults
+      for (var key in map) {
```

Comment:
See: https://github.com/gameclosure/devkit-core/blob/feature-individualCompile/src/build/common/spritesheetMapGenerator.js
Wanted to omit the generation of those defaults so that generation is faster and download is faster